specification/recoveryservices/resource-manager/Microsoft.RecoveryServices/stable/2016-06-01/vaults.json

```diff
@@ -589,6 +644,47 @@
       },
       "x-ms-azure-resource": true
     },
+    "ResourceNameAvailabilityInput": {
+      "description": "Resource information, as sent to the regional resource provider from Global RP.",
+      "type": "object",
+      "properties": {
+        "type": {
+          "type": "string"

```

Comment:
What exactly is type, would it make more sense to be an enum?  Perhaps a better description would help.

Reply:
It is the Resource Type: Microsoft.RecoveryServices/vaults. Added the description above in new PR: https://github.com/Azure/azure-rest-api-specs/pull/5496